README.md

```diff
@@ -38,9 +39,10 @@ Here's some of the features that Bloxstrap provides over the stock Roblox bootst
 * Lets you see what region your current server is located in
 * Lets you have multiple Roblox game instances open simultaneously
 
-All the available features are browsable through the Bloxstrap menu.
+All the available features are browsable through the Bloxstrap settings menu.
 
-Bloxstrap also only runs whenever necessary, so it doesn't stay running in the background when you're not playing.
+>[!TIP]
+>Bloxstrap also only runs whenever necessary, so it doesn't stay running in the background when you're not playing.
```

Comment:
This too, this is supposed to be touted more as a feature than a 'tip', as other modding tools that came before Bloxstrap **do** stay running in the background when you're not playing (rbxfpsunlocker, etc)

Reply:
maybe note will be better here?